hack/generated/controllers/suite_test.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	resources "github.com/Azure/k8s-infra/hack/generated/apis/microsoft.resources/v20200601"
+	"github.com/Azure/k8s-infra/hack/generated/controllers"
+	"github.com/Azure/k8s-infra/hack/generated/pkg/testcommon"
+)
+
+const (
+	TestRegion             = "westus"
+	TestNamespace          = "k8s-infra-test-ns"
+	DefaultResourceTimeout = 2 * time.Minute
+)
+
+var testContext ControllerTestContext
+
+type ControllerTestContext struct {
+	*testcommon.TestContext
+	SharedResourceGroup *resources.ResourceGroup
+}
+
+func setup() (ControllerTestContext, error) {
+	ctx := context.Background()
+	log.Println("Running test setup")
+
+	SetDefaultEventuallyTimeout(DefaultResourceTimeout)
+	SetDefaultEventuallyPollingInterval(5 * time.Second)
```

Comment:
can we qualify these with `gomega` in this file specifically? just for clarity

Reply:
Good idea